capstone/src/test/java/com/google/univiz/scorecard/ScorecardConverterTest.java

```diff
@@ -0,0 +1,52 @@
+package com.google.univiz.scorecard;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.base.Converter;
+import com.google.univiz.CarnegieSizeDegree;
+import com.google.univiz.CollegeData;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class ScorecardConverterTest {
```

Comment:
Can you also add a test to verify that the doBackwards [throws an exception](http://dr/corp/drive/folders/10GnLdUrSiLxh5oHQ5ZOEc9g-ymsqPwCc)?